sandstone/client/sandstone/components/updateservice/updateservice.js

```diff
@@ -0,0 +1,23 @@
+angular.module('sandstone.updateservice', [])
+.service('UpdateService', ['$rootScope', function($rootScope){
+    var ws = null;
+    var setUpWebsocket = function() {
+        // TODO find better way to create WS address
+        var websocketAddress = "ws://" + window.location.hostname + ":" + window.location.port + "/messages";
```

Comment:
Websocket protocol should be one of `ws` or `wss` depending on whether or not SSL is being used.